src/org/traccar/database/DataManager.java

```diff
@@ -106,19 +110,23 @@ private void initDatabase() throws Exception {
                 Class.forName(driver);
             }
 
-            ComboPooledDataSource ds = new ComboPooledDataSource();
-            ds.setDriverClass(config.getString("database.driver"));
-            ds.setJdbcUrl(config.getString("database.url"));
-            ds.setUser(config.getString("database.user"));
-            ds.setPassword(config.getString("database.password"));
-            ds.setIdleConnectionTestPeriod(600);
-            ds.setTestConnectionOnCheckin(true);
-            ds.setMaxStatementsPerConnection(config.getInteger("database.maxStatements"));
+            HikariConfig hConfig = new HikariConfig();
```

Comment:
Can you please rename it to `poolConfig` or `hikariConfig` or something like that.